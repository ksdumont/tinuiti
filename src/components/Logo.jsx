import React from 'react';
import { Link } from 'react-router-dom';

function Logo() {
  return (
    <div>
      <Link to="/">
        <img className="tinuiti-logo" src="https://tinuiti.com/wp-content/uploads/2019/05/logo.svg" alt="Tinuiti" />
      </Link>
    </div>
  );
}

export default Logo;
