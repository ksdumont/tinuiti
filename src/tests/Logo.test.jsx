import React from 'react';
import ReactDOM from 'react-dom';
import Logo from '../components/Logo';

import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

it('renders correctly', () => {
  const wrapper = shallow(<Logo />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
