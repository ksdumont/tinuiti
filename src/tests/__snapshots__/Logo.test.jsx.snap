// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <Link
    to="/"
  >
    <img
      alt="Tinuiti"
      className="tinuiti-logo"
      src="https://tinuiti.com/wp-content/uploads/2019/05/logo.svg"
    />
  </Link>
</div>
`;
